docs/tutorial/dbinit.rst

```diff
@@ -44,6 +44,9 @@ Within the function, you can then access :attr:`flask.g` and other things as
 you might expect.  When the script ends, the application context tears down
 and the database connection is released.
 
+.. note:: In order to use the newly created ``flask initdb`` command, the 
+    application must be re-installed using ``pip install -e .``
```

Comment:
Probably should come in the previous step. Also a ordering of all commands in the order they have to be typed is probably preferrable.